content/devops/foundational-short-course/2-why-is-devops-important/_index.md

```diff
@@ -0,0 +1,38 @@
+---
+content_type: topic
+ready: true
+title: Why is DevOps important?
+---
+
+Before DevOps, a major methodology of developing software was used: waterfall. It consists of breaking down the development process into sequential, linear tasks, executing one after another and never reviewing a task after it is completed. It flows steadily downwards, like a waterfall.
```

Comment:
Maybe worth mentioning Agile as well.

It almost sounds like DevOps is a project management methodology. Project management is a piece of the puzzle.

Reply:
I agree. Added information about Agile and the relationship between Agile and DevOps.